docs/platforms/unreal/configuration/debug-symbols/index.mdx

```diff
@@ -50,3 +50,21 @@ To upload debug symbols to Sentry manually, run `sentry-cli` through the command
 ```bash
 sentry-cli --auth-token ___ORG_AUTH_TOKEN___ debug-files upload --org ___ORG_SLUG___ --project ___PROJECT_SLUG___ PATH_TO_SYMBOLS
 ```
+
+## Debug symbols for macOS and iOS
+
+Sentry requires [_dSYM files_](/platforms/unreal/data-management/debug-files/file-formats/#mach-o-and-dsym) to fully symbolicate your stack traces for macOS/iOS games. To ensure that the debug symbols are generated and can be processed by `sentry-cli` during the execution of `PostBuildSteps`, add the following build settings to your project `.Target.cs` file:
```

Comment:
we're referring here to sentry-cli, but the plugin deals with it automatically, we could make that more clear
```suggestion
Sentry requires [_dSYM files_](/platforms/unreal/data-management/debug-files/file-formats/#mach-o-and-dsym) to fully symbolicate your stack traces for macOS/iOS games. To ensure that the debug symbols are generated and can be processed by the Sentry Unreal plugin during the execution of `PostBuildSteps`, add the following build settings to your project `.Target.cs` file:
```

perhaps avoid referring to Sentry CLI since might lead folks to looking at downloading/calling it

Reply:
Good point, I've commited the suggested change